pom.xml

```diff
@@ -27,7 +27,6 @@
         <module>ui.content</module>
         <module>it.tests</module>
         <module>it.launcher</module>
-    <module>configDS</module>
```

Comment:
why we remove this pls justify is it needed or just u added it by mistake ?

Reply:
fixed